src/main/java/com/food4thought/food4thought/controller/OrderController.java

```diff
@@ -20,12 +20,26 @@ public List<Order> getAllOrders() {
         return orderService.getAllOrders();
     }
 
-    @GetMapping("/active_orders")
+    @GetMapping("/orders/active")
     public List<Order> getActiveOrders() {
         return orderService.getActiveOrders();
     }
 
+    @PostMapping("/orders")
+    public ResponseEntity<Order> postOrder(@RequestBody Order order) throws Exception {
 
+        Order o = orderService.createOrder(order);
+        if (o == null) {
+            throw new Exception();
```

Comment:
please throw a more specific exception, e.g. a NullPointerException, or specify a custom exception like Food4ThoughtException that extends RuntimeException.